### PR TITLE
[CP] Add phone field to CP form

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: src/Entity/CitizenProject.php [hunk 1]
```diff
@@ -364,6 +350,10 @@ public function update(
         if (null === $this->postAddress || !$this->postAddress->equals($address)) {
             $this->postAddress = $address;
         }
+
+        if (!$this->phone->equals($phone)) {
```
Comment (B-Galati) [inline src/Entity/CitizenProject.php]: What is the usefulness of this ?

Comment (HeahDude) [reply src/Entity/CitizenProject.php]: Persisting a clone while letting an orphan behind.

File: src/Admin/CitizenProjectAdmin.php [hunk 2]
```diff
@@ -111,6 +111,12 @@ protected function configureShowFields(ShowMapper $showMapper)
                     'label' => 'Longitude',
                 ])
             ->end()
+            ->with('Phone', ['class' => 'col-md-5'])
```
Comment (NyKo24) [inline src/Admin/CitizenProjectAdmin.php]: Téléphone ?

Comment (illusionOfParadise) [reply src/Admin/CitizenProjectAdmin.php]: Changed

**Pull request merged**